doc/api/console.markdown

```diff
@@ -109,6 +109,43 @@ console.assert(false, 'Whoops %s', 'didn\'t work');
   // AssertionError: Whoops didn't work
 ```
 
+It is important to note that the `console.assert()` method in Node.js is
+implemented differently than the `console.assert()` method available in
+browsers. Specifically, in browsers, calling `console.assert()` with a falsy
+assertion will cause the `message` to be printed to the console without
+interrupting execution of subsequent code. In Node.js, however, a falsy
+assertion will cause an `AssertionError` to be thrown.
+
+Functionality approximating that implemented by browsers can be implemented
+by extending Node.js' `console` and overriding the `console.assert()` method.
+
+In the following example, a simple module is created that extends and overrides
+the default behavior of `console` in Node.js.
+
+```js
+'use strict';
+
+const myConsole = Object.setPrototypeOf({
```

Comment:
Not sure we should be encouraging such liberal use of setPrototypeOf. A lot of people have strong feelings against it and it sort of lumped in the language by mistake (it's a funny story really).

If we _are_ using it though, why not just:

    const myConsole = { __proto__: console,
                        assert: ... }